core/jvm/src/main/scala/scalaz/zio/blocking/Blocking.scala

```diff
@@ -52,6 +52,7 @@ object Blocking extends Serializable {
      * If the returned `IO` is interrupted, the blocked thread running the synchronous effect
      * will be interrupted via `Thread.interrupt`.
      */
+    @deprecated("use effectBlocking()", "1.0.0")
```

Comment:
Can you make this one delegate to the other one, to avoid duplicated code?

Reply:
Removed duplicate code by delegating to new function.